src/components/editor.rs

```diff
@@ -188,27 +191,6 @@ impl<'a, DB: Database + DatabaseQueries> Component<DB> for Editor<'a> {
           sender.send(Action::Query(self.textarea.lines().to_vec(), false))?;
         }
       },
-      Action::CopyData(data) => {
-        #[cfg(not(feature = "termux"))]
-        {
-          Clipboard::new().map_or_else(
-            |e| {
-              log::error!("{e:?}");
-            },
-            |mut clipboard| {
-              clipboard.set_text(data.clone()).unwrap_or_else(|e| {
-                log::error!("{e:?}");
-              })
-            },
-          );
-          // also set textarea buffer as a fallback
```

Comment:
i do want to keep on copying data to the textarea'a buffer as a fallback in case arboard fails, and also for termux since arboard is not supported. you should be able to add this back into the CopyData handler in app.rs if you make editor.textarea pub(crate) i think

Reply:
I missed the point that arboard is not supported in termux. Since copying to clipboard is implemented in app.rs will such a code be enough?

```
Action::CopyData(data) => {
    #[cfg(not(feature = "termux"))]
    {
      self.textarea.set_yank_text(data);
    }
    #[cfg(feature = "termux")]
    {
      self.textarea.set_yank_text(data);
    }
},
```

Or remove the attribute *feature = "termux"*, because there is a duplication of the method set_yank_text?

Reply:
yeah i agree, this should be enough, and you can remove the attribute feature = "termux" since it's the same in both cases in editor.rs

Reply:
> yeah i agree, this should be enough, and you can remove the attribute feature = "termux" since it's the same in both cases in editor.rs

done :)